packages/radio-player/demo/radio-player-controller.ts

```diff
@@ -45,7 +45,7 @@ export default class RadioPlayerController extends LitElement {
 
   private fileName = '';
 
-  private baseUrl = 'https://58-review-radio-arch-bsdafk.archive.org';
+  private baseUrl = 'https://archive.org';
```

Comment:
QUESTION: You don't want this a relative URL?

Reply:
This is for the demo app to test searching. I meant to change it awhile ago since that review app no longer exists.